test/parallel/test-benchmark-util.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+require('../common');
+
+const runBenchmark = require('../common/benchmark');
+
+runBenchmark('util', ['n=1', 'arguments=0']);
```

Comment:
Setting `arguments=0` would turn the `arg` in the test `undefined`. I believe we are good with `n=1` only.